pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>io.github.mivek</groupId>
     <artifactId>metarParser</artifactId>
-    <version>1.10.0</version>
+    <version>1.10.1</version>

```

Comment:
Can you revert this change?
The pom will be updated by travis CI.

Reply:
:+1: done!